### PR TITLE
Added Dev Guidelines to README

### PR DESCRIPTION
**Pull request by artcz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,3 +53,49 @@ CONTRIBUTING
...
+
+Development Guidelines
+----------------------
+
+To give you some direction of where we're going with the codebase, here's a
+short of list of things we have in mind for the near future.
+
+* Current codebase needs major refactorings and updates, and we really
+  appreciate all the help, but before doing anything big please talk to us
+  (EuroPython Web WG), so we can coordiante with other ongoing developments.
```
Comment (alexsavio) [inline README.md]: coordinate

File: README.md [hunk 2]
```diff
@@ -53,3 +53,49 @@ CONTRIBUTING
...
+  You can use github issues for that, or find us on our public telegram group
+  here -> https://t.me/sprintseuropythonsite
+
+* We currently have three major django apps – p3, assopy and conference. They
+  are here for historical reasons, and our plan forward is to slowly get rid of
+  p3 and assopy and replace all of them with just conference app. (This is very
+  long term plan)
+
+* We've chosen to go with an approach of rewriting the epcon 'in place', which
+  in plain english means we're adding new features and APIs (python functions
```
Comment (alexsavio) [inline README.md]: English

File: README.md [hunk 3]
```diff
@@ -53,3 +53,49 @@ CONTRIBUTING
...
+  and classes) within `conference/` app, and then slowly removing old pieces of
+  logic from old apps. See `conference/invoicing.py` for an example.
+
+* We use pytest and prefer the pytest tests over DjangoTestCase tests, however
+  both are fine. Similar as above, we have new `tests/` directory and we put
```
Comment (alexsavio) [inline README.md]: we have a new `tests/`

File: README.md [hunk 4]
```diff
@@ -53,3 +53,49 @@ CONTRIBUTING
...
+  all tests, organised by topic in those files. See `tests/test_invoicing.py`
```
Comment (alexsavio) [inline README.md]: organised by topic,

File: README.md [hunk 5]
```diff
@@ -53,3 +53,49 @@ CONTRIBUTING
...
+
+* We prefer integration tests (ie. using django test client) over unit tests,
+  particulary for old features/pieces of code. Given that our current test
+  coverage is lower than we'd like it to be, pull request that just add tests are
```
Comment (alexsavio) [inline README.md]: pull requests

File: README.md [hunk 6]
```diff
@@ -53,3 +53,49 @@ CONTRIBUTING
...
+* We also like WIP (Work In Progress) pull requests, and Proof-of-Concept
+  proposals. If you'd like to work on something, that may take a long time,
+  please open WIP PR (and add WIP to the title)
+
+* Branch names that start with `feature/` `bugfix/` `tests/` and have
+  descriptive names like `docs/update-readme-with-dev-guidelines` are
+  preferable to 'patch-1' ;)
+
+
+  help there's a list of issues, and you can submit your own as well, but
```
Comment (alexsavio) [inline README.md]: is this a mistake? copy from above?

Comment (artcz) [reply README.md]: yup

**Review by alexsavio — APPROVED** [review]
